### PR TITLE
feat(integrations): Add analytics tracking for integration events

### PR DESCRIPTION
**Pull request by macqueen**

### CODE REVIEW

File: src/sentry/models/integration.py [hunk 1]
```diff
@@ -31,6 +32,7 @@ class OrganizationIntegration(Model):
     organization = FlexibleForeignKey('sentry.Organization')
     integration = FlexibleForeignKey('sentry.Integration')
     config = EncryptedJsonField(default=lambda: {})
+
```
Comment (EvanPurkhiser) [inline src/sentry/models/integration.py]: Seems like an unrelated change? Did the linter do this?

**Review by evanpurkhiser — APPROVED** [review]
